pkg/permissions/scripts/getCallableFunctions.ts

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import path from 'path';
+import { ActionIdInfo, ContractActionIdData, getActionIdFunctions, safeReadJsonFile } from '../src/actionIds';
+import { getAccountLabel } from '../src/labelling';
+
+const funcsPath = path.join(__dirname, '../../deployments/action-ids/mainnet/action-ids.json');
+
+const actionIdFunctions = getActionIdFunctions(safeReadJsonFile<Record<string, ContractActionIdData>>(funcsPath));
+
+const specialCasedActionIds: Record<string, ActionIdInfo[]> = {
+  // Withdraw permission over BALTokenHolder for veBAL gauge.
+  '0x79922681fd17c90b4f3409d605f5b059ffcbcef7b5440321ae93b87f3b5c1c78': [
+    {
+      taskId: '20220325-bal-token-holder-factory',
+      contractName: 'BALTokenHolder',
+      signature: 'function withdrawFunds(address,uint256)',
+      useAdaptor: false,
+    },
+  ],
+};
+
+const ignoredActionIds: string[] = [
```

Comment:
These are network specific, right?

Reply:
I'm making this package be mainnet only for the time being we're mostly interested in it for the authorizer migration.